MOBILE_API.xml

```diff
@@ -891,6 +891,27 @@
             <description>Similar to VP8, but VP9 is customized for video resolutions beyond high-definition video (UHD) and also enables lossless compression.</description>
         </element>
     </enum>
+	
+    <enum name="AudioStreamingIndicator" since="5.0">
+        <element name="PLAY_PAUSE">
+            <description>
+                Default playback indicator.
+                By default the playback indicator should be PLAY_PAUSE when:
+                    - the media app is newly registered on the head unit (after RegisterAppInterface)
+                    - the media app was closed by the user (App enteres HMI_NONE)
```

Comment:
This should say "enters `HMI_NONE" (removing extra "e" and adding code blocks around "HMI_NONE")